cpp/src/io/orc/reader_impl.cu

```diff
@@ -1122,7 +1122,8 @@ table_with_metadata reader::impl::read(uint64_t skip_rows,
         CUDF_EXPECTS(not is_stripe_data_empty or stripe_info->indexLength == 0,
                      "Invalid index rowgroup stream data");
 
-        stripe_data.emplace_back(total_data_size, _stream);
+        stripe_data.emplace_back(
```

Comment:
Sorry, this should be fixed in https://github.com/rapidsai/cudf/pull/13586 but git decided to omit it somehow when I merged into https://github.com/rapidsai/cudf/pull/13396.
```suggestion
        // Buffer needs to be padded.
        // Required by `copy_uncompressed_kernel`.
        stripe_data.emplace_back(
```